stingray/powerspectrum.py

```diff
@@ -390,3 +389,41 @@ def _make_segment_spectrum(self, lc, segment_size):
             end_ind += nbins
 
         return power_all, nphots_all
+
+
+class DynamicalPowerspectrum(AveragedPowerspectrum):
+    def __init__(self, lc, segment_size, norm="frac"):
+        """
+        Parameters
+        ----------
+        lc : lightcurve.Lightcurve object
+            The time series of which the Dynamical powerspectrum is
+            to be calculated.
+
+        segment_size : float, default 1
+             Length of the segment of light curve, default value is 1 second.
+
+        norm: {"leahy" | "rms"}, optional, default "rms"
+            The normaliation of the periodogram to be used. Options are
+            "leahy" or "rms", default is "rms".
```

Comment:
Well yes, the normalizations should be the same for all of them and that's why I copied the docstring from there only. See [here](https://github.com/StingraySoftware/stingray/blob/master/stingray/powerspectrum.py#L125-L127). Maybe we should be making changes for all of them including here?